### PR TITLE
[quant][graphmode] Remove quantizeBias

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #26390 [quant][graph] PyTorch Graph Mode Quantization API
* #26389 [quant][graph] Add inplace argument to InsertObservers and InsertQuantDeQuant
* #26425 [quant][graphmode] Get scalar type from observer module
* #26415 [quant][graphmode] Remove quantization for bias in pattern
* #26414 [quant][graphmode] Fix quantized::linear QuantFusion patterns
* **#26388 [quant][graphmode] Remove quantizeBias**

Summary:
We don't need to quantize bias after https://github.com/pytorch/pytorch/pull/26057

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

Differential Revision: [D17465554](https://our.internmc.facebook.com/intern/diff/D17465554)

### CODE REVIEW

File: torch/csrc/jit/passes/quantization.cpp [hunk 1]
```diff
@@ -601,12 +568,7 @@ void InsertQuantDeQuantImpl(
             InsertQuantDeQuantImpl(m.value(), module_method_name);
           }
         }
-        if (v->node()->kind() == prim::GetAttr &&
-            v->node()->s(c10::attr::name) == "bias") {
-          qh.quantizeBias(v);
-        } else {
-          qh.quantizeTensor(v);
-        }
+        qh.quantizeTensor(v);
```
Comment (ZolotukhinM) [inline torch/csrc/jit/passes/quantization.cpp]: It seems that after this change we will be quantizing bias - we would just do that in the same manner as a usual tensor. Is it intended or do we want to skip bias tensors completely?

Comment (jerryzh168) [reply torch/csrc/jit/passes/quantization.cpp]: oh, I'll restore the check

Comment (jerryzh168) [reply torch/csrc/jit/passes/quantization.cpp]: to support this properly we'll probably need a value set as well.. but we can do this later

**Review by ZolotukhinM — APPROVED** [review]
👍

**Pull request closed without merging**